src/reducers/game.js

```diff
@@ -53,7 +53,13 @@ const game = handleActions(
     }),
 
     [actionTypes.FETCH_GIST_SUCCESS]: (state, action) =>
-      parseGistGame(action.payload)
+      parseGistGame(action.payload),
+
+    [actionTypes.SET_CALL_UNDER_MOUSE]: (state, { payload }) => ({
```

Comment:
Is there a need for a `CLEAR_CALL_UNDER_MOUSE`? Why not?

Reply:
I use calling setCallUnderMouse with null to clear it. I will change this to a specialized clear command